test/UnitTests/MSTest.CoreAdapter.Unit.Tests/Execution/LogMessageListenerTests.cs

```diff
@@ -82,20 +82,4 @@ public void DisposeShouldDisposeTraceListener()
     }
 
     #endregion
-
-    public void LogMessageListenerShouldCaptureLogMessagesInAllListeningScopes()
```

Comment:
Test was testing behavior of something used only in PS.Portable but never shipped.